app/styles/components/extras.scss

```diff
@@ -3,10 +3,17 @@ a.link-item {
   display: table;
 
   &.active {
-    background-color: rgba(0, 0, 0, .1);
+    background-color: 'rgba(0, 0, 0, 0.1)';
```

Comment:
Why?

Reply:
@iamareebjamal , there is a pre-commit hook that was not allowing leading zeros.

Reply:
Use .1 as it was before